doc/cheatsheet.md

```diff
@@ -0,0 +1,255 @@
+% djot cheatsheet
+
+## Basics
+
+| Markup | Result |
+| --- | --- |
+| `_italic_` or `{_italic_}` | *italic* |
+| `*bold*` or `{*bold*}` | **bold** |
+| `` `verbatim/code` `` | `verbatim/code` |
+| `H~2~O` | H~2~O |
+| `20^th^` | 20^th^ |
+| `{=highlighted=}` | <mark>highlighted</mark> |
+| `{+insert+}` | <ins>insert</ins> |
+| `{-delete-}` | <del>delete</del> |
+
+Use triple-backticks for fenced verbatim blocks:
+
+~~~~~~
+```
```

Comment:
AFAIK GFM supports only backtick codeblocks so you have to use backticks here (in this case more than 3).

Reply:
GFM supports tildes for fenced code too (like CommonMark)!
But the technique of using more backticks outside (in this case 4) than inside (3 here), I find prettiest.

Reply:
If you view the rendered file at https://github.com/uvtc/djot/blob/patch-1/doc/cheatsheet.md , it seems to work fine.